src/applications/personalization/profile360/components/MilitaryInformation.jsx

```diff
@@ -128,7 +131,7 @@ class MilitaryInformationContent extends React.Component {
             <a href="tel:18663632883" aria-label="1. 8 6 6. 3 6 3. 2 8 8 3.">
```

Comment:
Did we intend to replace this or was the addition of a new phone number intended?

Reply:
Updated! Thanks! 🙂